doc/source/whatsnew/v1.6.0.rst

```diff
@@ -168,7 +168,7 @@ Indexing
 
 Missing
 ^^^^^^^
--
+- Bug in :meth:`Index.equals` raising ``TypeError` when :class:`Index` consists of tuples that contain ``NA`` (:issue:`48446`)
```

Comment:
looks like this is not rendering correctly due to unbalanced backticks around TypeError.